mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-flex-service-subschema.json

```diff
@@ -39,12 +39,28 @@
                                 "Microsoft.DBforPostgreSQL",
                                 "Microsoft.DBforMariaDB"
```

Comment:
Change the type from `enum` to `string` and list the known values in `examples`?

Reply:
sounds good - I'll also implement better handling of unknown provider values on the service side